src/pybind/mgr/ssh/module.py

```diff
@@ -375,7 +375,7 @@ def _run_ceph_daemon(self, host, entity, command, args, stdin=None):
 
             out, err, code = remoto.process.check(
                 conn,
-                ['/usr/bin/python3', '-u'],
+                ['/usr/bin/python', '-u'],
```

Comment:
I guess this might not work, if there is only Python 3 installed on a system?

Reply:
Hmm, I just assumed a py3-only install would still have /usr/bin/python present.  I don't have any py3-only hosts around to check, though?

Reply:
Sigh, the [PEP](https://www.python.org/dev/peps/pep-0394/#id24) doesn't provide much insights. Looks like everything can happen.